### DISCUSSION TITLE
Updating the Already existing blog

### DISCUSSION THREAD

**Discussion by gautamsarawagi** [facebook/docusaurus#10433] [Q&A] [answered]
I just downloaded the default classic template just as mentioned in the [docs](https://docusaurus.io/docs/installation) here.

But when I went to one of the blogs and updated it. The UI got changed: 

![image](https://github.com/user-attachments/assets/31f697d0-ba97-4d78-aa4b-d64c5a4605a8)

Even I after reverting the changes the UI is not coming properly as for othere: 

![image](https://github.com/user-attachments/assets/09c4cb3a-f9a5-477e-85cb-1437c9748a7e)

Would really appreciate any help on this.

**Comment by slorber** [comment]
We don't know what change you made, and what is the result you expect, so we can't really help

The 2 screenshots do not show the same blog post 🤷‍♂️ 

If you want help, share a repro, or the code that leads to an unexpected result

---

On the first screenshot, most likely you did an update and your blog yaml front matter is no longer valid due to messing up with the `---` delimiters. I can only guess because you didn't share the updated code.

**Comment by gautamsarawagi** [comment]
I was just using the example code that we get in start: https://github.com/facebook/docusaurus/tree/2b28b06f67125872073d0609ddac1387d6c3bec4/examples/classic

**Comment by gautamsarawagi** [comment, marked as answer]
@slorber you were right I found that I was missing `---` somehow. Don't know why it was getting removed on editing always. But adding that cleared the issue. Thanks a lot. Closing this one.
